src/main/java/de/retest/recheck/persistence/bin/KryoPersistence.java

```diff
@@ -35,6 +38,17 @@
 
 	private static final String OLD_RECHECK_VERSION = "an old recheck version (pre 1.5.0)";
 
+	private static final Map<Class<?>, Integer> versions = createVersions();
```

Comment:
Maybe `compatibleVersions` or something like this? Just to make it easier to understand what these key-value pairs are for.